Source/Core/Core/Host.h

```diff
@@ -44,3 +44,4 @@ void Host_UpdateMainFrame();
 void Host_UpdateTitle(const std::string& title);
 void Host_YieldToUI();
 void Host_UpdateProgressDialog(const char* caption, int position, int total);
+void Host_PPCTitleChanged();
```

Comment:
Might as well call this `Host_TitleChanged` rather than `Host_PPCTitleChanged`.

Reply:
Done.